addons/website_jitsi/static/src/js/chat_room.js

```diff
@@ -29,6 +29,8 @@ publicWidget.registry.ChatRoom = publicWidget.Widget.extend({
         // default username for jitsi
         this.defaultUsername = this.$el.data('default-username') || false;
 
+        this.jitsiServer = this.$el.data('jitsi-server') || 'meet.jit.si';
```

Comment:
Could be great to add a docstring to start with some explanations of all parameters